e2e/cypress/integration/ui/common/user_settings.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('login the app', () => {
+    describe('load and update user preference', () => {
+        before(() => {
+            cy.visit('/');
+            cy.contains('Sign in as a local user').click({ force: true }).then(() => {
```

Comment:
We already have cypress spec files that contains this test cases and you need to just add those file 
Please follow this file:
https://github.com/chef/automate/blob/b8486c44deeb30362f8691fe1fa5f5025d60e042/e2e/cypress/integration/ui/infra-proxy/infra-roles.spec.ts#L21-L24

Reply:
done